config/docker/rustc-1.62.jinja2

```diff
@@ -0,0 +1,42 @@
+{% extends 'clang-15.jinja2' %}
+
+{% block packages %}
+{{ super() }}
+
+ENV RUST_VER=1.62.0
+ENV BINDGEN_VER=0.56.0
+
+ENV RUST_TRIPLE=rust-${RUST_VER}-x86_64-unknown-linux-gnu
+
+ENV CARGO_HOME=/usr/$RUST_TRIPLE/cargo
+
+ENV PATH=/usr/$RUST_TRIPLE/bin:${PATH}
+
+# fetch, validate and import the rust team key
+RUN wget https://static.rust-lang.org/rust-key.gpg.ascii
+RUN gpg --import --import-options show-only rust-key.gpg.ascii | \
+        grep -q 108F66205EAEB0AAA8DD5E1C85AB96E6FA1BE5FE
+RUN gpg --import rust-key.gpg.ascii
+
+# fetch, verify the toolchain
+RUN wget https://static.rust-lang.org/dist/${RUST_TRIPLE}.tar.gz
+RUN wget https://static.rust-lang.org/dist/${RUST_TRIPLE}.tar.gz.asc
+RUN gpg --verify ${RUST_TRIPLE}.tar.gz.asc ${RUST_TRIPLE}.tar.gz
```

Comment:
If the version needs to be fixed, then an easier/faster way is to download the `.tar.gz` and compare its sha256 against one stored here.

Reply:
The signature way should be fine since you were checking the full fingerprint above (I think it would require a preimage attack on SHA-1?), but yeah, using a hardcoded thing is simpler (though requires a bit more updating when changing the version, of course). For similar reasons I suggested the same for `tc-build`.

Also, speaking of simplicity: you can probably combine some of these steps into a single `RUN`, since I don't think it buys much having several layers per step?